src/Core/Cache/DefaultCacheFactory.php

```diff
@@ -114,20 +117,52 @@ protected function isPHPFilesSupported()
     }
 
     /**
-     * @param string $class
-     * @param array $args
-     * @return CacheInterface
+     * Creates an object with a PSR-16 interface, usually from a PSR-6 class name
+     *
+     * Quick explanation of caching standards:
+     * - Symfony cache implements the PSR-6 standard
+     * - Symfony provides adapters wrap a PSR-6 backend with a PSR-16 interface
+     * - Silverstripe uses the PSR-16 to interface with caches
+     * - Psr\SimpleCache\CacheInterface is the php interface of the PSR-16 standard
```

Comment:
```suggestion
     * - Psr\SimpleCache\CacheInterface is the php interface of the PSR-16 standard. All concrete cache classes Silverstripe code interacts with should implement this interface.
```
I think this is what is meant?

Reply:
Updated